source/designers-guide/google-pagespeed-best-practise/index.md

```diff
@@ -0,0 +1,83 @@
+
+* Install `optipng` on your server
+* Install `jpegtran` on your server
+* *Optional:* Run `sw:media:optimize` to optimize all existing images on the server
+
+### Thumbnails
+
+Talking about thumbnails, you may want more thumbnail sizes to give the browser more options to decide from. 
+
+**Example**
+
+An image is shown with a width of `201px` and your thumbnail sizes are `200x200` and `600x600`. The browser will then fetch the bigger `600x600` image, because the `200x200` would be too small, even if it's just 1 pixel. Google PageSpeed then complains about the requested image is too big for the given viewport.
+
+For this case, you should add more thumbnail sizes to provide the browser a more granular set of thumbnails to decide from.
+
+Keep in mind, that the more thumbnail sizes you define, the longer it will take to upload a picture to the media manager. In addition, an uploaded image requires more disk space since more thumbnail files will be generated.
+
+**Recommendation for the best score**
+
+* Generate thumbnails every 100px - 150px. `100x100`, `200x200`, `300x300`, `400x400`, ... 
+* All available thumbnail sizes are automatically provided in the view, so there is no need to modify template files. 
+
+### Responsive images
+
+Now that there are more thumbnails to choose from, the browser needs to know, how big the image that will be displayed is.
+
+**Recommendation for the best score**
+
+There is a guide on how to work with responsive image including the attributes `srcset` and `sizes`.
+
+Please refer to the [Responsive images](/designers-guide/responsive-images/) guide.
+
+## CSS
+
+Shopware delivers all theme styles in a single CSS file which will be loaded on page load. This works well as we only have 1 request to fetch all styles, but for Google PageSpeed, it is a nightmare as you will be faced with the `above-the-fold content` error message.
+
+Google PageSpeed expects that your site is fully renderable without additional network round trips. That means that your fully rendered site should not differ much from the initial render.
+
+**Recommendation for the best score**
+
+Google explains the concept of **Critical CSS** where the smallest needed part of CSS will be rendered into the DOM inside your `<head/>` element. The content of the style should contain everything to render the first visible part your website without loading any other remote styles.
+
+* Get the smallest needed part of your styles
+* Insert your critical CSS into the `<head/>` element
+
+## Javascript
+
+Since Shopware is compressing javascript when it compiles its themes you should always add your code to the `$javascript` array inside of the `theme.php` file. If this is for some reason not possible you should load your scripts asynchronously by using the `async` tag:
+```
+<script src="..." async>
+```
+Furthermore you should not load scripts synchronously which depend on jquery since this is loaded at the end of the site.
```

Comment:
Please rewrite this section for more details on when jQuery is loaded in Shopwarte and why it is necessary to load scripts async to not block the site rendering etc.